### PR TITLE
English translation of manual

### PR DESCRIPTION
**Pull request by jasperhabicht**

### CODE REVIEW

**Comment by jasperhabicht** [comment]
It would be great if the package owner could review my translation!

**Review by jasperhabicht — COMMENTED** [review]
Line 201 should probably be: `If the option \opt{c} is set or the option \opt{ratio} is set to a number other than $0$ or $1$, this option has no effect.`

File: wrapstuff-doc-en.tex [hunk 1]
```diff
@@ -0,0 +1,256 @@
...
+% This is file `wrapstuff-doc-en.tex`.
+% This file contains an English translation of the manual for the `wrapstuff` package (Copyright (C) 2022 by Qing Lee <sobenlee@gmail.com>).
+%
+% Translation by Jasper Habicht <mail@jasperhabicht.de>.
+% This file is under the same licence as the relevant files of the `wrapstuff` package.
+%
+\documentclass{ctxdoc}
+\usepackage{wrapstuff}
+\usepackage{graphicx}
+\usepackage{zhlipsum}
```
Comment (jasperhabicht) [inline wrapstuff-doc-en.tex:13]: Not needed.

File: wrapstuff-doc-en.tex [hunk 2]
```diff
@@ -0,0 +1,256 @@
...
+  section/format = \Large\bfseries\raggedright,
+  section/name   = {{},.\space},
+  figurename     = Figure,
+}
+
+\begin{document}
+
+  \changes{v0.1}{2022/07/05}{Initial version.}
+  \changes{v0.1}{2022/07/20}{First version submitted to CTAN.}
+  \changes{v0.2}{2022/07/21}{改进放置方式。}
```
Comment (jasperhabicht) [inline wrapstuff-doc-en.tex:70]: Change to: `\changes{v0.2}{2022/07/21}{Improvements of positioning mechanism.}`

File: wrapstuff-doc-en.tex [hunk 3]
```diff
@@ -0,0 +1,256 @@
...
+
+  \title{\bfseries The \pkg{wrapstuff} Package}
+  \author{LI Qing\thanks{Translation by Jasper Habicht <mail@jasperhabicht.de>}}
+  \date{\filedate\qquad\fileversion\thanks{\gitsha{\ExplFileVersion}.}}
```
Comment (Skillmon) [inline wrapstuff-doc-en.tex:74]: The resulting package date from this line is wrong, you're missing the `\GetFileId{wrapstuff.sty}` call from line 182 to correctly set the `\filedate` and `\fileversion`.

Comment (Skillmon) [reply wrapstuff-doc-en.tex:74]: Should be fixed by `\GetFileId` as well (as that also calls `\GetIdInfo` which should set `\ExplFileVersion`).

Comment (jasperhabicht) [reply wrapstuff-doc-en.tex:74]: @Skillmon It does not. I am actually not sure where the commit ID comes from as I could not find it anywhere. If I add `\GetFileId`, I get `-1` as result. It seems that the `ctxdoc` package looks for a file with `.id` ending and the current job name. The package maintainer probably has this file. As temporary fix, I would just hard code the commit ID.

File: wrapstuff-doc-en.tex [hunk 4]
```diff
@@ -42,17 +37,13 @@
 \tl_put_right:Nn \endExample
   { \endtrivlist }
 \SideBySideExampleSet { numbers = none }
-\makeatother
```
Comment (jasperhabicht) [inline wrapstuff-doc-en.tex]: Keep `\makeatother`.

**Comment by qinglee** [comment]
Sorry for the late reply. Thank you very much for your translation. It's fine with me.

**Pull request merged**